### PR TITLE
Moves get_client to _get_client and fixes calls to it in mcli.

### PR DESCRIPTION
**Pull request by ed-**
Changing it to _get_client obscures it from the "Available actions" output.

### CODE REVIEW

**Comment by rnirmal** [comment]
+1

**Comment by jcru** [comment]
+1

**Comment by cp16net** [comment]
i'd be happy with this working. +1

File: reddwarfclient/mcli.py [hunk 1]
```diff
@@ -129,6 +129,7 @@ def hwinfo(self):
     def diagnostic(self):
         """List diagnostic details about an instance."""
         self._require('id')
+        dbaas = self._get_client()
```
Comment (ed-) [inline reddwarfclient/mcli.py]: After the rebase I'm not certain about this, though _get_client has side effects that are probably desirable here.

**Pull request merged**